src/lock.ts

```diff
@@ -85,17 +130,40 @@ export class N9MongoLock {
 
 	/**
 	 * Release the lock
+	 * @param code: the code of the lock to be released
+	 * @param suffix : a key to identify the specific lock
 	 */
-	public async release(code: string): Promise<boolean> {
-		const ret: boolean = await new Promise<boolean>((resolve, reject) => {
-			this.lock.release(code, (err: any, ok: boolean) => {
-				if (err) return reject(err);
-				return resolve(ok);
-			});
-		});
-		// Wait to let enough time to someone else to pick the lock
-		await waitFor(this.options.n9MongoLockOptions.waitDurationMsMax + 5);
+	public async release(code: string, suffix?: string): Promise<boolean> {
+		const now = Date.now();
```

Comment:
should be moved to a private release function to be more clear

Reply:
Don't we need to access this function from other places ?